<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ED2-1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Arquivos.qrc">
    <normaloff>:/new/Arquivos/Arquivos/crf-flamengo-3-1.png</normaloff>:/new/Arquivos/Arquivos/crf-flamengo-3-1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;ESTRUTURA DE DADOS II - ATIVIDADE 04&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-60</y>
      <width>941</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Arquivos.qrc">:/new/Arquivos/Arquivos/Fundo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS GERADOS :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red;
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string>GERAR</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/gear-solid.svg</normaloff>:/new/Arquivos/Arquivos/gear-solid.svg</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-color: red;
border-radius: 10px; /* Raio para fazer o botão parecer redondo */
background-color: #ffffff; /* Cor de fundo não transparente */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-color: red;
border-radius: 10px; /* Raio para fazer o botão parecer redondo */
background-color: #ffffff; /* Cor de fundo não transparente */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ff0000;&quot;&gt;QUANTIDADE DE ELEMENTOS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditElementos">
       <property name="styleSheet">
        <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonBuscar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/magnifying-glass-solid.svg</normaloff>:/new/Arquivos/Arquivos/magnifying-glass-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;BUSCA SEQUENCIAL MELHORADO :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResultado">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px; 
color: red;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;VALOR :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;RESULTADO :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarSequencial">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-color: red;
border-radius: 10px; /* Raio para fazer o botão parecer redondo */
background-color: #ffffff; /* Cor de fundo não transparente */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;DADOS VETOR ORDENADO :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaOrdenado">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red;
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarOrdenado">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-color: red;
border-radius: 10px; /* Raio para fazer o botão parecer redondo */
background-color: #ffffff; /* Cor de fundo não transparente */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;BUSCA BINARIA :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValorBinaria">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;VALOR :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResultadoBinaria">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red; 
border-radius: 5px; 
color: red;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;RESULTADO :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBuscarBinario">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: red;
    border-radius: 10px;
    background-color: #ffffff;
}
QPushButton:pressed {
    border-color: blue;
    background-color: #cccccc;
}
</string>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/magnifying-glass-solid.svg</normaloff>:/new/Arquivos/Arquivos/magnifying-glass-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>640</y>
      <width>439</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;SÁVIO REIS DE LIMA GONÇALVES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarBinaria">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-color: red;
border-radius: 10px; /* Raio para fazer o botão parecer redondo */
background-color: #ffffff; /* Cor de fundo não transparente */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</normaloff>:/new/Arquivos/Arquivos/trash-solid.svg</iconset>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>textEditSaida</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButtonApagar</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>pushButtonBuscar</zorder>
   <zorder>label_7</zorder>
   <zorder>lineEditValor</zorder>
   <zorder>lineEditResultado</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButtonApagarSequencial</zorder>
   <zorder>label_5</zorder>
   <zorder>textEditSaidaOrdenado</zorder>
   <zorder>pushButtonApagarOrdenado</zorder>
   <zorder>label_10</zorder>
   <zorder>lineEditValorBinaria</zorder>
   <zorder>label_11</zorder>
   <zorder>lineEditResultadoBinaria</zorder>
   <zorder>label_12</zorder>
   <zorder>pushButtonBuscarBinario</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButtonApagarBinaria</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
